src/components/createBook/CreateBook.tsx

```diff
@@ -141,8 +141,8 @@ function CreateBook() {
                 getOptionValue={(option: AuthorResponse) => option.Id.toString()}
                 onChange={handleSelectedAuthorsChange}
               />
-              <button className='add-btn' onClick={() => setShow((s) => !s)}><AddIcon className = "icon" size={40} color="#fce4db" /></button>
-              { show && <CreateAuthor setIsAuthorsChanged={setIsAuthorsChanged} closeModal={() => setShow(false)}/>}
+              <button className='add-btn' onClick={() => setShowModal((s) => !s)}><AddIcon className = "icon" size={40} color="#fce4db" /></button>
```

Comment:
Avoid naming with one letter like `(s) => !s` 
While it doesn't make a huge difference, it reads better to me.